src/main/res/layout/layout_authentication.xml

```diff
@@ -26,9 +28,12 @@
 	android:paddingLeft="32dp"
 	android:paddingRight="32dp"
 	android:layout_width="match_parent"
-	android:layout_height="wrap_content">
+	android:layout_height="wrap_content"
+	android:descendantFocusability="beforeDescendants"
+	android:focusableInTouchMode="true">
 
 	<ImageView
+		android:id="@+id/logo"
```

Comment:
Please add this id to the landscape layout also! Currently the app is crashing in landscape mode!